core/qbft/qbft.go

```diff
@@ -229,6 +226,11 @@ func Run[I any, V comparable](ctx context.Context, d Definition[I, V], t Transpo
 				break
 			}
 
+			justified := isJustified(d, instance, msg)
+			if !justified {
```

Comment:
nit: can inline `if !isJustified {`

Also, we should probably still debug log this fact:
```
if !isJustified(d, instance, msg){
  d.LogUponRule(instance, process, round, msg, "unjust")
  break
}
```

Reply:
yes, I thought I was going to pass it to clasify but in the end is not necessary

Reply:
Done